custom_components/econet300/__init__.py

```diff
@@ -23,7 +25,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     cache = MemCache()
 
     try:
-        api = await make_api(hass, cache, entry.data)
+        data: dict[Any, Any] = dict(entry.data)
```

Comment:
This part should not be left as Any

There should be defined posible return Data types, for example dict[str | int | float, str | int | bool | None]

Reply:
change it to str
https://github.com/jontofront/ecoNET-300-Home-Assistant-Integration/pull/66/commits/9d8fd583ef2887ea09e6fd6b347aad27c8e7c51b